openapi2kong/oas3_testfiles/18-request-validator-plugin-path-params-outside-ops.expected.json

```diff
@@ -0,0 +1,70 @@
+{
+  "_format_version": "3.0",
+  "services": [
+    {
+      "host": "backend.com",
+      "id": "730d612d-914b-5fe8-8ead-e6aa654318ef",
+      "name": "example",
+      "path": "/path",
+      "plugins": [],
+      "port": 80,
+      "protocol": "http",
+      "routes": [
+        {
+          "id": "bee0be08-646a-562a-91b9-71737169585b",
+          "methods": [
+            "GET"
+          ],
+          "name": "example_test-common-param-common-param_get",
+          "paths": [
+            "~/test/common-param/(?\u003ccommon_param\u003e[^#?/]+)$"
```

Comment:
at some point the output switched to escaping the curly braces here, after I updated the underlying lib version. This script: https://github.com/Kong/go-apiops/blob/main/openapi2kong/oas3_testfiles/_fixup_expected.sh can be used to fix those.